src/store/Chat/actions.js

```diff
@@ -1,8 +1,10 @@
 import {
   CHAT_OPEN_CONVERSATION,
```

Comment:
Can we remove this constant and its associated actions, since I think CHAT_SET_ACTIVE accomplishes the same thing and is more accurate